src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -31,41 +36,48 @@
 from gsy_e.models.strategy.finite_power_plant import FinitePowerPlant # NOQA
 
 from gsy_e.models.leaves import Leaf # NOQA
-from gsy_e.models.leaves import *  # NOQA
-from gsy_framework.utils import convert_pendulum_to_str_in_dict, key_in_dict_and_not_none
+from gsy_e.models.leaves import *  # NOQA  # pylint: disable=wildcard-import
 
 
 class AreaEncoder(json.JSONEncoder):
```

Comment:
Nit: is this class only used for testing? I see it only being used in `area_to_string`, which is only called in tests.

Reply:
If this is the case, maybe it would make sense to move it inside the test module.

Reply:
I hesitate to move this in the test module because in the past it was used for serializing a running simulation. And I am fairly certain that we might need this functionality soon, therefore instead of hiding it inside the test module it makes more sense to have it together with the deserializer.